certora/specs/ghoVariableDebtToken.spec

```diff
@@ -643,3 +661,25 @@ rule burnAllDebtReturnsZeroDebt(address user) {
 	uint256 variableDebt_ = balanceOf(e, user);
     assert(variableDebt_ == 0);
 }
+
+
+
+
+rule mutant_11() {
+    env e;
+    address user; address onBehalfOf; uint256 amount; uint256 index;
+    //    require getUserCurrentIndex(onBehalfOf) <= index;
+    //require user ==1; require onBehalfOf==10;
+    //    require user != onBehalfOf;
+
+    mathint amountScaled = rayDiv(e,amount,index);
+    mathint prev_bal = scaledBalanceOf(e, onBehalfOf);
+    
+    mint(e,user,onBehalfOf,amount,index);
+
+    mathint after_bal = scaledBalanceOf(e, onBehalfOf);
+
+    assert (after_bal <= prev_bal + amountScaled);
```

Comment:
what about the range we talked about?
after_Bal >= prev_bal && after_Bal <= prev_bal + amountScaled

Reply:
I think that the lower bound is not correct. In the function _mintScaled there are situations where token are burnt.